Jellyfin.Networking/Manager/INetworkManager.cs

```diff
@@ -0,0 +1,231 @@
+#nullable enable
```

Comment:
This is already enabled in the csproj?

Reply:
Cause it's moved in PR2 into a nullable disabled project.